docker-compose-base.yml

```diff
@@ -23,8 +21,15 @@ server:
     - PYTHONDONTWRITEBYTECODE=1
     - DATABASE_URL=postgres://postgres@db/postgres
     - DEBUG=True
+    - INITIAL_ADMIN_USERNAME=admin
```

Comment:
So, `docker-compose.yml` will only be used for development. Values are hardcoded here to give a consistent dev setup for everyone. 

But, Elastic Beanstalk takes the place of Docker Compose in production & staging. So, something like [00-environment.setup-dist](https://github.com/mozilla/idea-town/pull/143/files#diff-bc50c16765dc9905b5e41c3d2fc5e1c1) will be used there.

We won't want to share any config values between dev/stage/prod, I don't think.